engine/src/main/scala/cromwell/webservice/routes/wes/Wes2CromwellInterface.scala

```diff
@@ -0,0 +1,31 @@
+package cromwell.webservice.routes.wes
+
+import akka.actor.{ActorRef}
+import akka.http.scaladsl.model.{HttpHeader}
+import akka.util.Timeout
+import com.typesafe.config.ConfigFactory
+import net.ceedubs.ficus.Ficus._
+import cromwell.services.metadata.MetadataService
+import cromwell.webservice.routes.MetadataRouteSupport.metadataQueryRequest
+
+import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.{ExecutionContext, Future}
+
+final class Wes2CromwellInterface()(implicit ec: ExecutionContext) {
+
+
+  implicit lazy val duration: FiniteDuration = ConfigFactory.load().as[FiniteDuration]("akka.http.server.request-timeout")
+  implicit lazy val timeout: Timeout = duration
+
+  def listRuns(pageSize: Option[Int], pageToken: Option[String], headers: List[HttpHeader], serviceRegistryActor: ActorRef): Future[WesResponse] = {
+    // FIXME: to handle - page_size, page_token
+    // FIXME: How to handle next_page_token in response?
```

Comment:
Are these fixmes inherited from the original wes2cromwell or are they new here? If they're new we should be sure to either fix them now, or (1) make a ticket to fix them and (2) add a release note saying that we have dropped this functionality (which can be removed if the ticket is completed before we do another release).

Since this is kind of equivalent to the `query` endpoint, and that already has a pagesize mechanism, I'd hope it isn't too hard to duplicate the functionality here

Reply:
These are inherited from the original wes2cromwell. It may be worth exploring once all the other endpoints have been migrated, if it isn't obvious on how to handle these before then.

Reply:
After a conversation with Chris (06.02.22), we both agreed that a separate ticket can be created to get this implemented correctly. As for this ticket, the endpoint functions as expected and there is no need to correct these now.